### DISCUSSION TITLE
Is it possible to implement a one step Arrhenius mechanism in PeleC?

### DISCUSSION THREAD

**Discussion by himcraft** [AMReX-Combustion/PeleC#887] [Q&A] [answered]
Hi all,

I noticed the question had been asked [here](https://github.com/AMReX-Combustion/PeleC/discussions/713#discussioncomment-8239862) but unanswered. So I would like to continue the topic: What should I add/modify if I want to implement a one-step Arrhenius mechanism in PeleC? 

Or should I post this question to the PelePhysics repo?

Many thanks!

**Comment by baperry2** [comment, marked as answer]
Assuming you have a Cantera or Chemkin mechanism file for your one-step chemistry, you can convert it to source code for use in Pele with the Ceptr utility in PelePhysics: https://amrex-combustion.github.io/PelePhysics/Ceptr.html

**Reply by himcraft** [reply]
Thank you! I will take a try.
